src/pilot/dell_nfv.py

```diff
@@ -211,62 +173,66 @@ def edit_environment_files(
             logger.info("Editing dell environment file")
             file_path = home_dir + '/pilot/templates/dell-environment.yaml'
             dpdk_file = home_dir + '/pilot/templates/neutron-ovs-dpdk.yaml'
+            cmds = []
             if not os.path.isfile(file_path):
                 raise Exception(
                     "The dell-environment.yaml file does not exist")
             if not os.path.isfile(dpdk_file):
                 raise Exception(
                     "The neutron-ovs-dpdk.yaml file does not exist")
-            cmds = ['sed -i "s|  # NovaSchedulerDefaultFilters|  ' +
-                    'NovaSchedulerDefaultFilters|" ' + file_path]
-            cmds.append(
-                'sed -i "s|dellnfv::hugepages::enable:.*'
-                '|dellnfv::hugepages::enable: ' +
-                str(enable_hugepage) +
-                '|" ' +
-                file_path)
-            cmds.append(
-                'sed -i "s|dellnfv::numa::enable:.*|dellnfv::numa::enable: ' +
-                str(enable_numa) +
-                '|" ' +
-                file_path)
-
+            if not ovs_dpdk:
```

Comment:
Why restricting this command to execute only if ovs_dpdk is enabled? Some of the filters included are needed to NUMA and Hugepages. NUMATopology and AggregateFilter are some of those.

Reply:
So but if none of the features are enabled, ie neither Hugepages, nor numa nor dpdk then default filters apply.

Reply:
Yes, This is same as way to old master's code 
https://github.com/uday344/JetPack/blob/ebaa424ce77b04edc1888dd71612f55283be1539/src/pilot/deploy-overcloud.py#L216